### DISCUSSION TITLE
[ENDPOINT]: Proposed design change for findAllUserFavoriteListings

### DISCUSSION THREAD

**Discussion by elizabethyalkut** [etsy/open-api#731] [Ideas]
### Is there an existing issue for this?

- [X] I have searched the existing tickets

### Current Endpoint Design

The `findAllUserFavoriteListings` is being deprecated in v3.

### Proposed Endpoint Design Change

I'd like to keep this endpoint in, as I'm working (slowly!) on an app to allow users to sort and filter their favorites, and won't be able to in the current design.

### Why are you proposing this change?

A possible use case for the app I'm creating is to find a favorited listings under or over a specific price point, which can't be done on the Etsy site currently. It would also be interesting to see if a user has favorited multiple listings from a given shop, or sort favorited listings by price.

**Comment by etsyachristensen** [comment]
@elizabethyalkut Unfortunately the user favorites functionality has been retired within the OpenAPI for Etsy. I don't think it will be resurrected, but I've added it to our discussion board to talk about. I wouldn't count on it making the system any time soon though.

**Comment by etsyachristensen** [comment]
Converting this to an idea discussion because we are closing the issues board soon.
